### ISSUE TITLE
Don't show user the auto-generated layer title in importer

### ISSUE THREAD

**Issue by jonpmarino** [MapStory/mapstory#838]
Instead of showing the user the auto-generated name we give to layers in importer, show an empty text box. Make the text box a bit wider too. Having the auto-generated text there encourages users to just leave it. Since the auto-generated title includes an underscore that currently hurts our search, this is particularly negative. The auto-generated layer name is only a technical requirement so doesn't need to be exposed to users at any point.

Also, update the text to say: 

"Giving your layer a descriptive title will make it easier for others to find. Remember to separate each word in your title with a space."

![screen shot 2017-07-17 at 3 04 08 pm](https://user-images.githubusercontent.com/2573761/28287598-ecc35c7a-6b01-11e7-8878-27eada984ad5.jpg)

**Comment by emilyashley** [comment]
Yes! I'm all about more intentional naming. 

However, we might want to display the file name from the upload somewhere on this modal (but outside of the text form) for the user to reference while creating their StoryLayer title.

**Comment by emilyashley** [comment]
Question from @Coop56:

> Importer has this importer.js file - https://github.com/GeoNode/django-osgeo-importer/blob/master/osgeo_importer/static/osgeo_importer/importer.js - I know we overwrite the HTML templates from Importer currently.  Do you know if we're overwriting that file as well or just using theirs?


### Braindump: 
We load most of the importer JS scripts into the browser via our Site Scripts template  [here](https://github.com/MapStory/mapstory/blob/master/mapstory/templates/_site_scripts.html#L77). Performance note: since the modals for Importer are part of the header menu options on every page we load these scripts (8 files) on every page.

That template tag `{% include osgeo_importer/_importer_scripts.html %}` contains:

```
/static/osgeo_importer/js/angular-resource.js
/static/osgeo_importer/js/underscore-min.js
/static/osgeo_importer/js/ui-bootstrap-tpls-0.14.2.min.js
/static/osgeo_importer/js/angular-cookies.js
/static/osgeo_importer/js/angular-wizard.min.js
/static/osgeo_importer/factories.js
/static/osgeo_importer/js/angular-file-upload.min.js
/static/osgeo_importer/importer.js
```
and I'm pretty sure the template for that tag is NOT in mapstory, but in osgeo_importer so we can't control the bundling easy.


We provide overwrites for some of Importer's html templates & wizards in the `mapstory/mapstory/static/mapstory/partials/` directory [here](https://github.com/MapStory/mapstory/tree/master/mapstory/static/mapstory/partials)

- createLayerModal
- upload
- uploadWizard

As of the time of writing these notes, I haven't confirmed that we're USING them, but they're there.


#### Of note: we do provide our own Angular controllers for:
    
- *Create Layer* and *Create Layer Modal* [here](https://github.com/MapStory/mapstory/blob/master/mapstory/static/mapstory/js/src/create-layer.controller.js)

- *Profile Uploads* Tab [here](https://github.com/MapStory/mapstory/blob/master/mapstory/static/mapstory/js/src/profile.uploads.controller.js)

And those are tied into the mapstory angular module right now and loaded with mapstory.js

Long story short, I don't believe we're overwriting `importer.js`, it's loaded on every page by including a django template that we also don't house.
